google_compute_engine/accounts/accounts_utils.py

```diff
@@ -150,7 +152,7 @@ def _UpdateUserGroups(self, user, groups):
     """
     groups = ','.join(groups)
     self.logger.debug('Updating user %s with groups %s.', user, groups)
-    command = ['usermod', '-G', groups, user]
+    command = self.usermod_cmd.format(user=user, groups=groups)
     try:
       subprocess.check_call(command)
```

Comment:
check_call accepts a list or a string. A list is usually preferable as it deals with escaping and quoting of arguments, but we are not doing anything complex, I just forget to add shell=True in the call as recommended by the docs

From the docs https://docs.python.org/2/library/subprocess.html:

> args is required for all calls and should be a string, or a sequence of program arguments. Providing a sequence of arguments is generally preferred, as it allows the module to take care of any required escaping and quoting of arguments (e.g. to permit spaces in file names). If passing a single string, either shell must be True (see below) or else the string must simply name the program to be executed without specifying any arguments.